VS.Data.Sqlite/VS.Data.Sqlite.csproj

```diff
@@ -85,7 +85,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Data.Sqlite.Core">
-      <Version>6.0.1</Version>
+      <Version>6.0.7</Version>
```

Comment:
We'll probably need to update to a 7.0 preview for this to work. My [super hacky code](https://github.com/bricelam/VS.Data.Sqlite/blob/498500184460fe98bc81ef96b3435c8ad11e62f1/VS.Data.Sqlite/SqliteProviderObjectFactory.cs#L21-L28) doesn't handle binding redirects.

Reply:
Fixed and verified that it works!